Frameworks/AddressBook/ABAddressBook.mm

```diff
@@ -16,44 +16,87 @@
 
 #import <AddressBook/ABAddressBook.h>
 
-#import <StubReturn.h>
 #import "AssertARCEnabled.h"
+#import <StubReturn.h>
+
+#import "_ABAddressBook.h"
+
+#import "UWP/InteropBase.h"
+#import "UWP/WindowsDevicesEnumeration.h"
+#import "initguid.h"
+
+// GUID to check Windows Contacts permissions.
+DEFINE_GUID(_ABAddressBookContactsGUID, 0x7D7E8402, 0x7C54, 0x4821, 0xA3, 0x4E, 0xAE, 0xEF, 0xD6, 0x2D, 0xED, 0x93);
```

Comment:
>DEFINE_GUID [](start = 0, length = 11)

Is this GUID available from some Windows header, rather than redefining it ourselves?